build-system/app.js

```diff
@@ -486,19 +511,31 @@ app.use('/impression-proxy/', function(req, res) {
 // Example:
 // http://localhost:8000/max/s/www.washingtonpost.com/amphtml/news/post-politics/wp/2016/02/21/bernie-sanders-says-lower-turnout-contributed-to-his-nevada-loss-to-hillary-clinton/
 app.use('/max/', function(req, res) {
+  if (mode != 'max') {
```

Comment:
How about we just rename it `/proxy/` or `/cdn/`? Then, we'll just need one and we can remove `/min/`